src/main/kotlin/net/onelitefeather/clipboardconnect/setup/DurationPrompt.kt

```diff
@@ -12,13 +12,23 @@ import kotlin.time.Duration
  */
 class DurationPrompt : StringPrompt() {
     override fun getPromptText(context: ConversationContext): Component {
+        if (context.getSessionData("parse-error")  != null) {
+            context.allSessionData.remove("parse-error")
+            return MiniMessage.miniMessage().deserialize("<red>Something went wrong parsing the duration string, please follow the format!<br><green>How long do you want to keep the clipboards in the Redis memory? <gray>(example: <gold>6h = 6 hours, 2d = 2 days<gray>)")
```

Comment:
Something went wrong while parsing the duration string
(add "while" for better understanding)